maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -149,6 +136,17 @@ def get_header(self, step):
         if exclusive:
             modified_header.append("#SBATCH --exclusive")
 
+        # If neither Procs nor Nodes exist, throw an error
+        procs = resources.get("procs")
+        nodes = resources.get("nodes")
+        if procs == None and nodes == None:
```

Comment:
This condition won't work because `procs` and `nodes` are never set to `None`. You also can't achieve that with the `get` call because they will exist in the resources as empty strings (`""`). Try this conditional: `if not procs and not nodes:` -- this clause will work because Python equates `""` to a "False" value in an `if` clause.